### ISSUE TITLE
[QUESTION] Can a multilabel model predict label combinations not seen during training?

### ISSUE THREAD

**Issue by enescicek** [lightly-ai/lightly-train#943]
Hi,
I have a question about multilabel image classification with LightlyTrain.
My dataset contains the following labels:

- `cat`
- `dog`
- `rabbit`

The training CSV contains examples with labels such as:

```
image_path,label
train/cat/001.jpg,"cat"
train/dog/002.jpg,"dog"
train/rabbit/003.jpg,"rabbit"
train/cat_dog/004.jpg,"cat,dog"
```

However, the training dataset does not contain any image labeled with `"cat,rabbit"`.
I train the model using:

```
lightly_train.train_image_classification(
    classification_task="multilabel",
    ...
)
```

Can the trained model still predict both `cat` and `rabbit` for an image containing them, even though this label combination was never present during training?
Are the class outputs learned independently, or could the absence of this combination cause the model to learn that `cat` and `rabbit` are mutually exclusive?
Would you recommend including examples of every important label combination in the training set?
Thank you!

@liopeer

**Comment by liopeer** [comment]
Hi @enescicek,
Classes are learned independently, so it should largely work. However, it might be that your model gets biased towards "class A and B" always appear together, but "B and C" never appear together. That is something you should take into account.

**Comment by enescicek** [comment]
On the Lightly Train documentation page, there is a section in the data part like this:

```
image_path,label
train/cat/img1.jpg,"cat"
train/cat/img2.jpg,"cat,dog"
train/car/img1.jpg,"car"
```

The image that contains both cat and dog is stored in the cat folder. The question that came to my mind here is:

Can an image that contains both cat and dog be stored in both the cat and dog folders at the same time?

My other question is the following:

```
import lightly_train

if __name__ == "__main__":
    lightly_train.train_image_classification(
        out="out/my_experiment",
        model="dinov3/vitt16",
        data={
            "train": "my_data_dir/train.csv",
            "val": "my_data_dir/val.csv",
            "classes": {
                0: "cat",
                1: "car",
                2: "dog",
                # ...
            },
            # Optional, classes that are in the dataset but should be ignored during
            # training.
            "ignore_classes": [0],
        },
    )
```

Do we specify the classes individually in the `classes` parameter?

For example, if I also want to use images that contain both cat and car at the same time, do I need to specify this in `classes` as well, like 3: "Cat-and-car"?
